src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/rightcurly/InputRightCurlyTestAloneOrSingleline.java

```diff
@@ -244,4 +244,14 @@ public void codeAfterLastRightCurly() {
 
     interface Interface2
     { int i = 1; public void meth1(); }
+
+    private void foo28() {
+        if (true) { return; } else { } if (false) { // 2 violations
+        }
+    }
+
+    private void foo31() {
+        do { var2 ++; }
+        while (var2 < 15); while (var1 < 10) { var1++; }
```

Comment:
This mutation kills https://github.com/checkstyle/checkstyle/pull/12618#discussion_r1066187654 .
It needs to be funky like this since it checks line positions.

Reply:
https://github.com/checkstyle/checkstyle/actions/runs/3886458243/jobs/6631584122#step:6:416
This isn't working with this PR. I will have to relook at this.

Reply:
I was made a mistake in my testing and this doesn't help pitest.